src/odemis/util/test/img_test.py

```diff
@@ -1181,8 +1183,838 @@ def test_rotate_img_metadata(self):
+        """
+        # force ValueError by deleting PIXEL_SIZE from the MD
+        del self.tile_collection[0][0].metadata[model.MD_PIXEL_SIZE]
+        result_shape = tuple(len(self.tile_collection) * elem for elem in self.size)
+
+        with self.assertRaises(ValueError):
+            img.getCenterOfTiles(self.tile_collection, result_shape)
+
+
+class TestGetBoundingBox(unittest.TestCase):
+
+    def setUp(self):
+        # create a 5D size and metadata with necessary items
+        self.size = (9, 10, 13, 400, 600)
+        self.md = {
+            model.MD_DIMS: 'CTZYX',
+            model.MD_PIXEL_SIZE: (1e-6, 1e-6),
+            model.MD_POS: (0.95, 0.07)
+        }
+
+    def test_get_bounding_box_normal(self):
+        """
+        Verify that using a 5D shaped DA will return the right bounding box.
+        """
+        # create a DA with random values
+        new_da = numpy.random.random(self.size)
+        new_da = model.DataArray(new_da, self.md)
+
+        output_pos = img.getBoundingBox(new_da)
+
+        # set the positions to check for
+        pos_x = new_da.metadata[model.MD_POS][0]
+        pos_y = new_da.metadata[model.MD_POS][1]
+
+        # check if the min position of the bounding box is the same as half the 
+        # size of the DA calculating from the middle point on the 0-axis downwards
+        expected_pos_x_min = pos_x - self.size[-1] * self.md[model.MD_PIXEL_SIZE][0] / 2
+        self.assertEqual(output_pos[0], expected_pos_x_min)
+
+        # check if the max position of the bounding box is the same as half the
+        # size of the DA calculating from the middle point on the 0-axis upwards
+        expected_pos_x_max = pos_x + self.size[-1] * self.md[model.MD_PIXEL_SIZE][0] / 2
+        self.assertEqual(output_pos[2], expected_pos_x_max)
+
+        # check if the min position of the bounding box is the same as half the 
+        # size of the DA calculating from the middle point on the 1-axis downwards
+        expected_pos_y_min = pos_y - self.size[-2] * self.md[model.MD_PIXEL_SIZE][1] / 2
+        self.assertEqual(output_pos[1], expected_pos_y_min)
+
+        # check if the max position of the bounding box is the same as half the
+        # size of the DA calculating from the middle point on the 1-axis upwards
+        expected_pos_y_max = pos_y + self.size[-2] * self.md[model.MD_PIXEL_SIZE][1] / 2
+        self.assertEqual(output_pos[3], expected_pos_y_max)
+
+    def test_get_bounding_box_incomplete_metadata(self):
+        """
+        Verify that removing PIXEL_SIZE in the metadate DA will fail and a LookupError exception is raised.
+        """
+        # create a DA with random values
+        new_da = numpy.random.random(self.size)
+        new_da = model.DataArray(new_da, self.md)
+
+        # remove the DIMS key and check if a return value is passed and no exception is raised
+        del self.md[model.MD_DIMS]
```

Comment:
This does the same, but I feel it'd be more explicit:
```suggestion
        del new_da.metadata[model.MD_DIMS]
```